components/helm-broker/internal/bundle/repository_config.go

```diff
@@ -0,0 +1,36 @@
+package bundle
+
+import (
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+// RepositoryConfig provides configuration for HTTP Repository.
+type RepositoryConfig struct {
+	BaseURL string `json:"baseUrl" valid:"required"`
```

Comment:
IMO it is not anymore a BaseURL, it can be but not always it is. My suggestion is to name it just `URL`.

Reply:
I will change it with next commit during upgrade component version along with values.yaml in core.